index.css

```diff
@@ -110,3 +110,283 @@ main {
+  gap: 88px;
+}
+
+.cards {
+  background: url(IMG/header-background-mobile2.png);
+  height: 596px;
+  width: 313px;
+  background-repeat: no-repeat;
+  border-radius: 10px;
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  margin-left: 24px;
+  padding-left: 16px;
+  display: flex;
+  padding-top: 16px;
+  flex-direction: column;
+}
+
+.card1 {
+  padding-top: 16px;
+  flex-direction: column;
+}
+
+.snapshot1 {
+  width: 295px;
+  height: 220px;
+  justify-items: center;
+}
+
+.tonic {
+  margin-top: 12px;
+  font-family: "Poppins", sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-bottom: 0;
+}
+
+.tonic:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.ulc1 {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 91px;
+  padding-left: 0;
+  list-style: none;
+  margin-bottom: 0;
+}
+
+.li1 a {
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+}
+
+.li2 a {
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #7a869a;
+}
+
+.li3 a {
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #7a869a;
+}
+
+.li1 a:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.li2 a:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.li3 a:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.par3 {
+  font-family: "Poppins", sans-serif;
+  flex-direction: column;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+}
+
+.par3:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.ulc2 {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  padding-left: 0;
+  padding-right: 90px;
+}
+
+.lis {
+  text-decoration: none;
+  padding: 12px;
+  background-color: #ebebff;
+  border-radius: 5px;
+  font-family: "Poppins", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.lis:hover {
+  text-decoration: underline;
+  text-decoration-color: #6070ff;
+}
+
+.btn1 {
+  padding: 12px;
+  text-decoration: none;
+  background-color: #ebebff;
+  width: 110px;
+  border: solid 1px #6070ff;
+  border-radius: 10px;
+  font-family: "Poppins", sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly add a margin-top on the button to separate it from the list of technology. As it may improve the look of the app
<img width="330" alt="Screen Shot 2022-03-18 at 13 41 41" src="https://user-images.githubusercontent.com/28307960/158997978-7b5d588d-fa9e-4531-88c4-f38be1fcb46c.png">